config/module.config.php

```diff
@@ -38,11 +38,12 @@
         ],
     ],
     'service_manager' => [
-        'invokables' => [
-            'BsbFlysystemManager' => 'BsbFlysystem\Service\FilesystemManager',
+        'aliases' => [
+            'BsbFlysystem\Service\FilesystemManager' => 'BsbFlysystemManager',
```

Comment:
I prefer that too, but that's not supported in PHP5.4 Definitely will be using that once 5.4 support is dropped